wasp/draw565.py

```diff
@@ -329,6 +329,15 @@ def string(self, s, x, y, width=None, right=False):
         if width:
             self.fill(bg, x, y, rightpad, h)
 
+    def char_size(self, char):
+        """Get the size of a specific character in the current font.
+
+        :param char: Character to get the size of.
+        :returns     Tuple with  (width, height) of character.
+        """
+        (_, h, w) = self._font.get_ch(char)
+        return (w, h)
```

Comment:
Height will always be constant so it would be better to implement this as separate get_char_width() and get_char_height() methods since in almost all cases the h value will be discarded (therefore allocating a tuple to hold it should be avoided).

Reply:
good point - changes incoming.